[2025-03-16T20:04:00.926+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: passanger_traffic.creare1 manual__2025-03-16T20:03:53.876965+00:00 [queued]>
[2025-03-16T20:04:00.933+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: passanger_traffic.creare1 manual__2025-03-16T20:03:53.876965+00:00 [queued]>
[2025-03-16T20:04:00.933+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-16T20:04:00.940+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): creare1> on 2025-03-16 20:03:53.876965+00:00
[2025-03-16T20:04:00.942+0000] {standard_task_runner.py:57} INFO - Started process 196 to run task
[2025-03-16T20:04:00.944+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'passanger_traffic', 'creare1', 'manual__2025-03-16T20:03:53.876965+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/passenger_traffic_dag.py', '--cfg-path', '/tmp/tmpzhgfga0h']
[2025-03-16T20:04:00.946+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask creare1
[2025-03-16T20:04:00.957+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-16T20:04:00.981+0000] {task_command.py:415} INFO - Running <TaskInstance: passanger_traffic.creare1 manual__2025-03-16T20:03:53.876965+00:00 [running]> on host 16ae585d7ab0
[2025-03-16T20:04:01.034+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='passanger_traffic' AIRFLOW_CTX_TASK_ID='creare1' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T20:03:53.876965+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T20:03:53.876965+00:00'
[2025-03-16T20:04:01.036+0000] {sql.py:274} INFO - Executing: 
    CREATE SCHEMA IF NOT EXISTS presentation;
[2025-03-16T20:04:01.042+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:04:01.047+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:04:01.052+0000] {sql.py:418} INFO - Running statement: 
    CREATE SCHEMA IF NOT EXISTS presentation;, parameters: None
[2025-03-16T20:04:01.060+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=passanger_traffic, task_id=creare1, execution_date=20250316T200353, start_date=20250316T200400, end_date=20250316T200401
[2025-03-16T20:04:01.084+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-16T20:04:01.100+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
